node/utils/utils.ts

```diff
@@ -1,3 +1,33 @@
+import { AuthenticationError, ForbiddenError, UserInputError } from '@vtex/api'
+import type { AxiosError } from 'axios'
+
+export const statusToError = (e: AxiosError) => {
+  if (!e.response) {
+    throw e
+  }
+
+  const {
+    response: { status },
+  } = e
+
+  switch (status) {
+    case 401: {
+      throw new AuthenticationError(e)
+    }
+
+    case 403: {
+      throw new ForbiddenError(e)
+    }
+
+    case 400: {
+      throw new UserInputError(e)
+    }
+
+    default:
+      break
```

Comment:
I think it's better to throw the original error here. Otherwise, any error not listed here will not be thrown.